src/main/scala/me/rexim/morganey/module/Module.scala

```diff
@@ -1,13 +1,35 @@
 package me.rexim.morganey.module
 
-import me.rexim.morganey.ast.MorganeyBinding
+import me.rexim.morganey.ast._
+import me.rexim.morganey.syntax._
+import me.rexim.morganey.util._
 import scala.util._
 
+import java.net.URL
+
 class Module(modulePath: ModulePath, classLoader: ClassLoader = Module.getClass.getClassLoader) {
   def canonicalPath: String =
     modulePath.asCanonicalPath.path
 
-  // TODO: Implement Module.{bindings, dependencies}
-  def bindings: Try[Set[MorganeyBinding]] = ???
-  def dependencies: Try[Set[Module]] = ???
+  private def nodes: Try[List[MorganeyNode]] = {
```

Comment:
Yeah, I think it is, but my trail of thoughts was the following: `lazy val` essentially caches the module content. If we change it the module object will still return the "cached" values and to reload the module we have to create the same instance again.